### PR TITLE
add ally skills prospectus

### PR DESCRIPTION
**Pull request by yochannah**
I think we need to add the link to the menu too. and maybe a Table of Contents?

### CODE REVIEW

File: consulting.md [hunk 1]
```diff
@@ -8,3 +8,100 @@ photos:
...
+- [Exercise: Set Ally Skills Goals](https://docs.google.com/document/d/1k3iBHkPZ18NJQ5zrdWB5mERPXopm4trJeVIBNSQqhbY/copy)
+- [Terminology](https://docs.google.com/document/d/1iGj11dxJJiAjpa_-Q9CZf1H4Htu-ojo2WUJYfVhK3NM/copy)
+- [Workshop Slides](https://docs.google.com/presentation/d/1qOiSO6dcgYHxrVLm4ShvGbr8KgBNyYKT/edit#slide=id.p1)
+
+[Download a copy of the course description](https://docs.google.com/document/d/1BfwarY5zR5jFBI_UemDfQdmcHS4nQViar95W2yt8fx8/edit?usp=sharing)
+
+### OLS Ally Skills Workshops: Organisation Guide
+
+The Open Life Science (OLS) Ally Skills workshop trainers are trained from the Frame Shift Consulting and training materials provided by them are built on the CC BY-SA 4.0 Licensed resources provided by Frame Shift Consulting (© 2016 - 2020), Dr Sheila Addison (© 2016) and The Ada Initiative (© 2011 - 2015). To learn more about Open Life Science, our open science training and mentoring programme and how to collaborate with us, please visit https://openlifesci.org. For any enquiry and details please email team@openlifesci.org.
```
Comment (NPDebs) [inline consulting.md:38]: "For any enquiry and details, please..."

Tinny tiny comma missing.

File: consulting.md [hunk 2]
```diff
@@ -8,3 +8,100 @@ photos:
...
+
+[Download a copy of the course description](https://docs.google.com/document/d/1BfwarY5zR5jFBI_UemDfQdmcHS4nQViar95W2yt8fx8/edit?usp=sharing)
+
+### OLS Ally Skills Workshops: Organisation Guide
+
+The Open Life Science (OLS) Ally Skills workshop trainers are trained from the Frame Shift Consulting and training materials provided by them are built on the CC BY-SA 4.0 Licensed resources provided by Frame Shift Consulting (© 2016 - 2020), Dr Sheila Addison (© 2016) and The Ada Initiative (© 2011 - 2015). To learn more about Open Life Science, our open science training and mentoring programme and how to collaborate with us, please visit https://openlifesci.org. For any enquiry and details please email team@openlifesci.org. 
+
+### About Ally Skills
+
+Often, when we witness oppression, we are so busy being shocked that we can't quickly react. Other times we have plenty of time to react but can’t figure out what to do, or we do something but it turns out to be more harmful than helpful. This is true even for people who have a lot of experience and education in supporting members of marginalised groups in their communities.
```
Comment (NPDebs) [inline consulting.md:42]: Same here.
"Other times, we have..."

File: consulting.md [hunk 3]
```diff
@@ -8,3 +8,100 @@ photos:
...
+
+### Aim
+
+OLS Ally Skills Workshops aim to equip participants with actionable, practical skills to use their societal advantages to support others in their workplace and communities, thus creating and contributing to a more inclusive workplace. 
+
+Diverse perspectives and collaboration drive better research and innovation. Meaningful collaborations require careful thinking about pathways and potential barriers to participation. If your suggestions to a project get continuously ignored, or if key decisions are made in conversations that you have no access to, chances are you will stop contributing to that project. For science to be truly collaborative and accessible, we should learn to recognise inequities in science and act to change that. It is important to support researchers in embedding ethical values in scientific projects and discuss the societal impacts of their work. OLS Ally Skills Workshops facilitate such discussions where we can explore our privileges as well as harms in different scenarios.
```
Comment (NPDebs) [inline consulting.md:50]: "...where we can explore our privileges, as well as..."

File: consulting.md [hunk 4]
```diff
@@ -8,3 +8,100 @@ photos:
...
+
+The scope of the workshop does not include teaching members of marginalised groups how to respond to oppression targeting themselves.
+
+### Workshop Format, Content & Logistics
+
+OLS Ally Skills workshops are offered to external organisations and collaborators for a fee. Please email the OLS team (team@openlifesci.org) to express your interest to host and request a quote. Since the OLS team members manage two 16-week long training and mentoring cohorts each year, it is important to get in touch with them several weeks in advance to share your plan and identify potential dates for the workshops.
+
+Customisation of our workshops: Our training is designed for researchers and scientific communities. We are happy to discuss if specific customisation in our workshop materials might be required for external organisations to represent their workspace culture. 
+
+The workshop is delivered in English. Nonetheless, we are very interested and welcome the possibility to work with a sign language interpreter or translator, please discuss this further with the OLS team.
```
Comment (NPDebs) [inline consulting.md:69]: "...interpreter or translator. Please discuss..."

File: consulting.md [hunk 5]
```diff
@@ -8,3 +8,100 @@ photos:
...
+
+- Duration of each workshop is 3 - 3.5 hours with breaks 
+- The workshop will be hosted online via Zoom. This will not be recorded or broadcasted
+- 2-3 trainers and co-facilitators will be available (trainers are trained by the Frameshift Consulting Train-the-trainers programme).
+- 20 participants make an ideal size for a single online session to allow safe space and open discussion. 
+- Workshop format will combine training, scenario-based group discussions and share outs. 
+- We use live transcription to enhance the accessibility of our workshop. This means that participants can see an automated live transcript (provided by otter.ai) during presentations and discussions held in the main room.
+- Discussion sessions are facilitated in spoken and written form allowing active contribution by all participants. 
+- Slides and handouts can be shared ahead of the workshop to participants who may have to miss parts of the workshop.
+- Different perspectives and experiences of participants enrich the discussion overall, however, participants will not be forced to share their thoughts.
```
Comment (NPDebs) [inline consulting.md:81]: "...enrich the discussion overall. However, participants..."

File: consulting.md [hunk 6]
```diff
@@ -8,3 +8,100 @@ photos:
...
+
+After the workshop, we would like to understand the impact of the workshop as well as gather any feedback from participants about the workshop-style and content. Please clarify with the OLS team in advance if this will be handled by your organisation or the OLS team directly.
+
+#### Logistics for the external organisations:
+
+Here are a few more considerations to help plan this workshop at your organisation:
+
+- Ally skills training needs to be opt-in registration - it’s fine to encourage people to come, but important that no one reluctant or ambivalent is asked to attend. This is to ensure that the session remains productive and on-target, as well as ensuring people can state their expectations from the workshop in the registration form. 
+- Please let the OLS team know if you expect more than 20 members.
+- For the session to be effective we recommend no more than three levels of hierarchy attending the workshop, for example, employees, managers, and their managers. Ideally, one level only is better.
```
Comment (NPDebs) [inline consulting.md:99]: "...attending the workshop. For example, employees..."

**Review by NPDebs — CHANGES_REQUESTED** [review]
This is a fine piece of writing @yochannah. 
I've suggested a view adjustments. I hope you don't mind.

File: consulting.md [hunk 7]
```diff
@@ -8,3 +8,100 @@ photos:
...
+- Please let the OLS team know if you expect more than 20 members.
+- For the session to be effective we recommend no more than three levels of hierarchy attending the workshop, for example, employees, managers, and their managers. Ideally, one level only is better. 
+- We recommend organisations hosting the OLS Ally Skills workshop create a registration page that requires interested members to state their personal motivation to attend this workshop. The OLS team will be happy to provide a dedicated Eventbrite page for handling registration.
+- Participants should have access to internet bandwidth for video conferencing. We recommend that the event organisers check if their members may require any audiovisual equipment that can be arranged for them in advance.
+
+### References:
+- Testimonial by OLS Ally Skills workshop participants: https://openworking.wordpress.com/2020/12/07/ally-skills-workshop/
+- OLS post about a previous workshop: https://openlifesci.org/posts/2020/10/05/ally-skills-training/
```
Comment (bebatut) [inline consulting.md:105]:
```suggestion
- [OLS post about a previous workshop]({% link _posts/2020-10-05-ally-skills-training.md %})
```

File: consulting.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (bebatut) [reply consulting.md:38]: It is redundant with the introduction paragraph before

File: consulting.md [hunk 8]
```diff
@@ -8,3 +8,100 @@ photos:
...
+
+Here are a few more considerations to help plan this workshop at your organisation:
+
+- Ally skills training needs to be opt-in registration - it’s fine to encourage people to come, but important that no one reluctant or ambivalent is asked to attend. This is to ensure that the session remains productive and on-target, as well as ensuring people can state their expectations from the workshop in the registration form. 
+- Please let the OLS team know if you expect more than 20 members.
+- For the session to be effective we recommend no more than three levels of hierarchy attending the workshop, for example, employees, managers, and their managers. Ideally, one level only is better. 
+- We recommend organisations hosting the OLS Ally Skills workshop create a registration page that requires interested members to state their personal motivation to attend this workshop. The OLS team will be happy to provide a dedicated Eventbrite page for handling registration.
+- Participants should have access to internet bandwidth for video conferencing. We recommend that the event organisers check if their members may require any audiovisual equipment that can be arranged for them in advance.
```
Comment (bebatut) [inline consulting.md:101]:
```suggestion
- Participants should have access to internet bandwidth for video conferencing.

  We recommend that the event organisers check if their members may require any audiovisual equipment that can be arranged for them in advance.
```

File: consulting.md [hunk 9]
```diff
@@ -8,3 +8,100 @@ photos:
...
+
+### References:
+- Testimonial by OLS Ally Skills workshop participants: https://openworking.wordpress.com/2020/12/07/ally-skills-workshop/
+- OLS post about a previous workshop: https://openlifesci.org/posts/2020/10/05/ally-skills-training/
+- Report by OLS describing why we discuss Ally Skills in science: https://www.software.ac.uk/blog/2021-07-05-why-we-discuss-ally-skills-open-science
```
Comment (bebatut) [inline consulting.md:106]:
```suggestion
- [Report by OLS describing why we discuss Ally Skills in science](https://www.software.ac.uk/blog/2021-07-05-why-we-discuss-ally-skills-open-science)
```

**Comment by yochannah** [comment]
@bebatut I think I've merged your changes - okay to merge?

File: consulting.md [hunk 10]
```diff
@@ -8,3 +8,100 @@ photos:
...
+Multiple OLS community members are trained facilitators. We offer ad-hoc in-house workshops (starting at £2,000 GBP for universities) as well as anyone-may-register workshops. Visit Eventbrite to find the next dates available to you: [OLS Ally Skills Eventbrite](https://www.eventbrite.com/cc/ally-skills-1223279) or contact team@openlifesci.org for a bespoke quote. 
+
+Ally Skills Materials:  
+
+- [Exercise: Identify your Power and Privilege](https://docs.google.com/document/d/1xlRf6265EovRoolY1DFLE4N7N_8jajNvDS9YT1_Nc_I/copy)
+- [Exercise: Set Ally Skills Goals](https://docs.google.com/document/d/1k3iBHkPZ18NJQ5zrdWB5mERPXopm4trJeVIBNSQqhbY/copy)
+- [Terminology](https://docs.google.com/document/d/1iGj11dxJJiAjpa_-Q9CZf1H4Htu-ojo2WUJYfVhK3NM/copy)
+- [Workshop Slides](https://docs.google.com/presentation/d/1qOiSO6dcgYHxrVLm4ShvGbr8KgBNyYKT/edit#slide=id.p1)
+
+[Download a copy of the course description](https://docs.google.com/document/d/1BfwarY5zR5jFBI_UemDfQdmcHS4nQViar95W2yt8fx8/edit?usp=sharing)
```
[the remark this answers is not in the document]
Comment (yochannah) [reply consulting.md]: done!

**Comment by bebatut** [comment]
Should we make that visible from the navbar?

**Review by bebatut — APPROVED** [review]

**Pull request merged**